Sources/Liquid/Variable.swift

```diff
@@ -36,8 +36,24 @@ struct Variable {
 	
 	private func parseFilterArgs(_ parser: Parser) -> [Expression] {
 		var args: [Expression] = []
-		
```

Comment:
We can add a `var kwargs: [String: Expression] = [:]` here where we'll capture those keyed args